lxc/list.go

```diff
@@ -53,12 +55,24 @@ A single keyword like "web" which will list any instance with a name starting by
 A regular expression on the instance name. (e.g. .*web.*01$).
 A key/value pair referring to a configuration item. For those, the
 namespace can be abbreviated to the smallest unambiguous identifier.
+A key/value pair where the key is a shorthand. Multiple values must be delimited by ','. Available shorthands:
+  - type={'CONTAINER','VIRTUAL-MACHINE'}
+  - status={'RUNNING','STOPPED','FROZEN'}
```

Comment:
That's an incomplete list, maybe best to just put a generic description?

Reply:
I think a list of some examples would be very useful, maybe add `e.g.`, `a.i.`, `among others` or `et al.` instead?